packages/stream-model-instance-handler/src/model-instance-document-handler.ts

```diff
@@ -268,7 +281,7 @@ export class ModelInstanceDocumentHandler implements StreamHandler<ModelInstance
 
     // Now validate the relations
     await this._validateRelationsContent(ceramic, model, content)
-    if (!genesis && payload) {
+    if (!genesis && payload && !enforceImmutableFields) {
```

Comment:
this name is backwards now.  When the value is true, you don't enforce the immutable fields.  Can we either invert how the boolean is used so it matches the name, or change the name to `skipImmutableFieldsCheck` or something like that?

Reply:
I renamed it to `skipImmutableFieldsCheck`